### ISSUE TITLE
zstyle ":history-search-multi-word" highlight-color not used

### ISSUE THREAD

**Issue by vtnpp** [zdharma/history-search-multi-word#22]
- `$ zsh --version`:

```
zsh 5.7.1 (x86_64-pc-linux-gnu)
```

- Terminal: konsole 19.12.1-1

- Relevant `.zshrc` parts:

```
zstyle ":history-search-multi-word" highlight-color 'fg=yellow,bold'
zstyle ":plugin:history-search-multi-word" synhl 'true'
zstyle ':plugin:history-search-multi-word' active 'bold,underline'
zstyle ':plugin:history-search-multi-word' check-paths 'true'
zstyle ':plugin:history-search-multi-word' clear-on-cancel 'false'
zstyle ":history-search-multi-word" page-size '11'

typeset -gA HSMW_HIGHLIGHT_STYLES
HSMW_HIGHLIGHT_STYLES[alias]='bg=057,fg=228'
HSMW_HIGHLIGHT_STYLES[command]='fg=077'
HSMW_HIGHLIGHT_STYLES[commandseparator]='fg=207'
HSMW_HIGHLIGHT_STYLES[double-hyphen-option]='fg=039'
HSMW_HIGHLIGHT_STYLES[path]='fg=167,underline'
HSMW_HIGHLIGHT_STYLES[single-hyphen-option]='fg=039'

[...]

zinit light zdharma/history-search-multi-word
```

All settings work fine, except `zstyle ":history-search-multi-word" highlight-color 'fg=yellow,bold'`.

![Screenshot_2020-01-21_21-03-09](https://user-images.githubusercontent.com/59181811/72839117-ed74ca00-3c91-11ea-96d5-e4629491e81e.png)

As you can see, `less` appears with blue background and white foreground instead of yellow foreground and bold text.

What am I doing wrong?

**Comment by psprint** [comment]
After I've pasted the snippet into the shell I did get the expected behavior. Does it work for you too after pasting, maybe?

**Comment by vtnpp** [comment]
Indeed, it works in the current session if I run `zstyle ":history-search-multi-word" highlight-color 'fg=yellow,bold'` into the shell.

**Comment by vtnpp** [comment]
Fixed after updating to [d343cd0](https://github.com/zdharma/history-search-multi-word/commit/d343cd0d7f7bfb0fc204ce1b91db2b32fd06af7c)

Thank you!

**Comment by psprint** [comment]
n/p! Thanks for the report.
